src/otx/algorithms/common/adapters/mmcv/ops/multi_scale_deformable_attn_pytorch.py

```diff
@@ -78,7 +84,18 @@ def _custom_grid_sample(im: torch.Tensor, grid: torch.Tensor, align_corners: boo
     Returns:
         torch.Tensor: A tensor with sampled points, shape (N, C, Hg, Wg)
     """
-    device = im.device
+    ori_device = im.device
+
+    if ori_device != "cpu":
```

Comment:
Will that change the behavior on CUDA?

Reply:
No, this code is executed only if CUDA isn't used.